ui/tutorial-renderer/src/main/res/layout/activity_optin_onboarding.xml

```diff
@@ -0,0 +1,16 @@
+<?xml version="1.0" encoding="utf-8"?>
+<androidx.constraintlayout.widget.ConstraintLayout xmlns:android="http://schemas.android.com/apk/res/android"
+    xmlns:app="http://schemas.android.com/apk/res-auto"
+    android:layout_width="match_parent"
+    android:layout_height="match_parent">
```

Comment:
again, what's the point of the `ConstraintLayout` here?

Reply:
I will need it later when I add the indicator.